tools/linter_lib/custom_check.py

```diff
@@ -483,6 +483,7 @@ def build_custom_checkers(by_lang):
              ('scripts/lib/zulip_tools.py', '# We need sudo here, since the path will be under /srv/ in the'),
              ('scripts/lib/zulip_tools.py', 'subprocess.check_call(["sudo", "/bin/bash", "-c",'),
              ('scripts/lib/zulip_tools.py', 'subprocess.check_call(["sudo", "rm", "-rf", directory])'),
```

Comment:
We might be able to remove these older linter errors by using `run_as_root` as well?